plugins/autocast/package.json

```diff
@@ -42,15 +42,10 @@
   "dependencies": {
     "@flatfile/api": "^1.7.4",
     "@flatfile/hooks": "^1.3.2",
-    "@flatfile/listener": "^1.0.1",
-    "@flatfile/plugin-record-hook": "^1.4.4",
     "@flatfile/util-common": "^1.0.0"
   },
-  "devDependencies": {
-    "@rollup/plugin-commonjs": "^25.0.7",
-    "@rollup/plugin-json": "^6.0.1",
-    "@rollup/plugin-node-resolve": "^15.2.3",
-    "@rollup/plugin-typescript": "^11.1.5",
-    "rollup": "^4.3.0"
+  "peerDependencies": {
+    "@flatfile/plugin-record-hook": "^1.4.4",
```

Comment:
Without the `@flatfile/plugin-record-hook` as a `peerDependencies`, two versions of the recordHook are generated (`const BulkRecordHook = ...` and `const dist_BulkRecordHook = ...`) causing the `event.fetch error:  Error: HTTP error! Status: 400` error.